addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4799,6 +4799,7 @@ registry.ReplaceMedia = SnippetOptionWidget.extend({
         }
         linkEl.setAttribute('href', url);
         this.rerender = true;
+        this.$target.trigger('href_changed');
```

Comment:
Can you explain to me what this is for?
I see that you also do 
```js
this.$target.on('href_changed.link_popover', (e) => {
            // Do not change shown/hidden state.
            if (popoverShown) {
                this._loadAsyncLinkPreview();
            }
});
```
So it's for the preview in the popover (title, description and favicon) right ?
I tested on a runbot (15.0) and I have already those informations in the popover. So I don't understand :|

Reply:
I added this, so that when you modify the URL on the options side, it gets reflected in the popup preview.
E.g.:
- add link on image
- set URL to 'odoo.com'
- click on image => 'odoo.com' shows up in the popup
- edit URL, set to 'google.com'
- click on image => without this change the popup still shows 'odoo.com'